assets/css/content/general.css

```diff
@@ -34,6 +34,17 @@
 .content-inner .heading-with-actions.top-heading .icon-action {
   font-size: 1.2rem;
 }
+/* When content is wide enough, ensure action icon is to the right of, not below, heading */
+@container content (width > 600px) {
+  .content-inner .heading-with-actions.top-heading {
+    flex-wrap: nowrap;
+    gap: 32px;
+    align-items: flex-start;
+    .icon-action {
```

Comment:
I think this is invalid, as we are not using a preprocessor, so I want ahead and remove it, as it seems not necessary. Please double check.

Reply:
Nested selectors are now supported in CSS. This ensures the icon isn't too close to the text.

Reply:
Correcting my edited comment: the padding-top is necessary for vertical alignment.

Reply:
(When the heading runs into two lines or more.)

Reply:
Oh, fantastic, I will add it back! esbuild is complaining, so we may need to either update it or move it to the top. Thanks!

Reply:
@DavidOliver this seems to do the trick: https://github.com/elixir-lang/ex_doc/commit/b0a2412936379aca6662dd9d270501c7dc1b2ad9

Reply:
Great - thanks!